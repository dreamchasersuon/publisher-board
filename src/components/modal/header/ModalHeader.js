import React from 'react';
import PropTypes from 'prop-types';
import './ModaHeader.css';

export default function ModalHeader({ title }) {
  return (
    <header className="Modal-header">
      <h2>{title}</h2>
    </header>
  );
}

ModalHeader.propTypes = {
  title: PropTypes.string,
};
