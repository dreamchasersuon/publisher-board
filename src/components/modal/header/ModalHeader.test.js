import React from 'react';
import renderer from 'react-test-renderer';
import ModalHeader from './ModalHeader';

test('renders modal header with given title', () => {
  const tree = renderer.create(<ModalHeader title="Update User" />).toJSON();
  expect(tree).toMatchSnapshot();
});
