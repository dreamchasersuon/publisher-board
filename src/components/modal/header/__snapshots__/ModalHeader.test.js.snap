// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders modal header with given title 1`] = `
<header
  className="Modal-header"
>
  <h2>
    Update User
  </h2>
</header>
`;
